scripts/getTestenvProperties.sh

```diff
@@ -91,7 +91,8 @@ getTestenvProperties() {
 		fi
 
 		# append the info into $OUTPUT_FILE
-		{	echo "${REPO_NAME}_REPO=$(git remote show origin -n | grep -Po '(?<=Fetch URL: ).*')";
+		{
+			echo "${REPO_NAME}_REPO=$(git remote show origin -n | awk '/Fetch URL:/{print $3}')";
```

Comment:
I'd add a comment as to why awk.

Reply:
@karianna Hmmm I'm not sure that's necessary - why would you add a comment on why you're using any standard shell utility? It's just the right standard tool for the job. IMHO such a comment would only make sense when looking at this code change. Unclear on how a meaningful comment fo rthe casual reader who hadn't seen this PR could be written, but happy to take a suggestion.

Reply:
awk is the more prevalent utility, present in a known/stable form on all platforms we must test on, unlike as we just learned grep, which apparently is not available in a standard version/form across all platforms that we test on.  Agree that we do not need to explain the use of awk in a comment in the code, do not think it adds to the readability of the code.